syntaxes/GDShader.tmLanguage.json

```diff
@@ -341,12 +341,26 @@
             "match": "\\b[a-zA-Z_]\\w*(?=(?:\\s|/\\*(?:\\*(?!/)|[^*])*\\*/)*[(])"
         },
         "swizzling": {
-            "name": "variable.other.property.gdshader",
-            "match": "(?:[.]\\s*)(?:[xyzw]{2,4}|[rgba]{2,4}|[stpq]{2,4})\\b"
+            "match": "([.])\\s*([xyzw]{2,4}|[rgba]{2,4}|[stpq]{2,4})\\b",
+            "captures": {
+                "1": {
+                    "name": "punctuation.accessor.gdshader"
+                },
+                "2": {
+                    "name": "variable.other.property.gdshader"
+                }
+            }
         },
         "identifierField": {
-            "name": "entity.name.variable.field.gdshader",
-            "match": "(?:[.]\\s*)[a-zA-Z_]\\w*\\b"
+            "match": "([.])\\s*([a-zA-Z_]+)\\w*\\b",
```

Comment:
I have no idea what I was thinking when I added those parens. 

My only defense is that I try to avoid using `\w` or other regex special characters because I have to use regexes across a bunch of different platforms, and I don't trust the implementations to have the same definitions, or myself to remember the differences.